src/core/org/apache/jmeter/gui/SimpleCompoundEdit.java

```diff
@@ -0,0 +1,14 @@
+package org.apache.jmeter.gui;
+
+import javax.swing.undo.CompoundEdit;
+
+public class SimpleCompoundEdit extends CompoundEdit {
+
+    public boolean isEmpty() {
+        return size() == 0;
```

Comment:
My reasoning was that `isEmpty()` can be implemented more efficient than `size() == 0`